core/reporter-api/src/main/java/org/arquillian/reporter/api/builder/report/AbstractReportBuilder.java

```diff
@@ -55,6 +55,20 @@ public BUILDERTYPE addEntry(String entry) {
         return (BUILDERTYPE) this;
     }
 
+
+    @Override
+    public BUILDERTYPE addEntries(List<?> entries) {
+        for (Object entry : entries) {
+            if (entry instanceof Entry) {
+                addEntry((Entry) entry);
+            } else if (entry instanceof String) {
+                addEntry((String) entry);
+            }
```

Comment:
I've got a few notes:
1) we are using Java 8 in this project, so please use stream instead of `for` statements
2) if there is another (easy) way of implementing it, please avoid using `instanceof` as it may have a performance impact (mainly in for cycles)
3) the type of the method parameter `List<?>` is too general. What if you put some object there that is neither an Entry nor a String; then it won't be added into the list of entries at all (without any warning). 
4) what I would do is that I would create two methods  - one that would take a list of strings and a second one that would take a list of entries.
5) if you adding such a methods into the API, consider also using an array (not only the list) so the developer can write:
`addEntries("first entry", "section entry")`
without putting it into the list